packages/flagship/src/lib/ios.ts

```diff
@@ -21,11 +24,8 @@ export function bundleId(configuration: Config): void {
 
   helpers.logInfo(`updating iOS bundle id`);
 
-  fs.update(
-    path.ios.infoPlistPath(configuration),
-    /<key>CFBundleIdentifier<\/key>\s+<string>[^<]+<\/string>/,
-    `<key>CFBundleIdentifier</key><string>${bundleId}</string>`
-  );
+  exec(`plutil -replace CFBundleIdentifier -string ${bundleId} ${path.ios.infoPlistPath(configuration)}`);
```

Comment:
Instead of using `execSync` and passing unsantizied inputs, we should use `execFileSync` so the `bundleId` can be escaped properly. The risk with this approach is a malicious Flagship project could pass in shell code in the bundleId that would be blindly executed.

Change this to something like
```typescript
execFileSync('plutil', ['-replace', 'CFBundleIdentifier', '-string', bundleId, path.ios.infoPlistPath(configuration)])
```

Reply:
done